packages/app-bridge/src/types/index.ts

```diff
@@ -35,3 +35,4 @@ export * from './Template';
 export * from './Terrific';
 export * from './Topic';
 export * from './User';
+export * from './PlatformAppPublicMethods';
```

Comment:
```suggestion
export * from './PlatformAppMethods';
```
No mention of public or private required here